app/controllers/api/v1/admin/admin_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Api::V1
+  class Admin::AdminController < ApiController
```

Comment:
Curious to learn why Api::V1 is set as a module but Admin::AdminController is namespaced with ::? What goes in to deciding which is styled as which?

Reply:
Good question... I didn't think about it that much.

https://stackoverflow.com/a/7821792/1949363 was a good reminder just now that the two styles are not identical but in this case it doesn't matter much.

I think the best I have is, it looks more "natural" this way to me but that's not a great reason...